Products/CMFPlone/URLTool.py

```diff
@@ -14,26 +15,10 @@ class URLTool(PloneBaseTool, BaseTool):
     security = ClassSecurityInfo()
     toolicon = "skins/plone_images/link_icon.png"
 
-    @security.public
```

Comment:
Do we still need a security declaration?

Reply:
I wondered that too, and wanted to add `security.declarePublic("isURLInPortal")`, because I think `@security.public` can only work on a method.  But without any security declaration I added this Script Python in the ZMI and I could call it just fine, even as anonymous user:

```
urls = ["http://localhost:8080/Plone", "https://maurits.vanrees.org"]
for url in urls:
    ip = context.portal_url.isURLInPortal(url)
    print(f"{ip} {url}")

return printed
```